schemas/2015-08-01/Microsoft.Automation.json

```diff
@@ -0,0 +1,145 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2016-06-01/Microsoft.Automation.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Automation",
+    "description": "Microsoft Automation Resource Types",
+    "definitions": {
+        "automationAccounts": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "enum": [
+                        "Microsoft.Automation/automationAccounts"
+                    ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-01-01-preview"
+                    ]
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "sku": {
+                            "type": "object",
+                            "properties": {
+                                "family": {
+                                    "type": "string",
+                                    "description": "Automation account sku family."
+                                },
+                                "name": {
+                                    "enum": [
+                                        "Free",
+                                        "Basic"
+                                    ],
+                                    "description": "Automation account sku name."
+                                },
+                                "capacity": {
+                                    "type": "integer",
+                                    "description": "Automation account sku capacity."
+                                }
+                            },
+                            "description": "Automation account sku.",
+                            "required": [ "name" ]
+                        }
+                    },
+                    "required": [ "sku" ]
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Represents the plan associated with the account."
+                },
+                "location": {
```

Comment:
no need to define "location" and "tags" because all ARM resource has these two properties defined, in deploymentTemplate.json.